src/status_im/subs/wallet/wallet.cljs

```diff
@@ -332,6 +337,19 @@
  (fn [accounts]
    (remove :watch-only? accounts)))
 
+(rf/reg-sub
+ :wallet/accounts-with-current-asset
+ :<- [:wallet/accounts-without-watched-accounts]
+ :<- [:wallet/wallet-send-token-symbol]
+ :<- [:wallet/wallet-send-token]
+ (fn [[accounts token-symbol token]]
```

Comment:
Hey guys, historically, I find myself having to comment too frequently on PRs about missing unit tests for layer-3 subs. This recommendation is in our guidelines and I believe we should be more strict about following it (reasons below).

> Test [layer-3 subscriptions](https://day8.github.io/re-frame/subscriptions/) by
actually subscribing to them, so reframe's signal graph gets validated too.
>
> -- https://github.com/status-im/status-mobile/blob/634dd747681b14c44d3ec8d13d7b44ca251480b0/doc/new-guidelines.md#subscription-tests

These tests are powerful because we test using the subscription machinery on purpose instead of just testing the sub handler. Our macro `deftest-sub` helps with that. This means that if any of the input subscriptions, say `:wallet/wallet-send-token` change, the output of its calculation will be picked up by the tests of this sub `:wallet/accounts-with-current-asset`.

This means that by covering layer-3 subs with unit-tests, we are covering the subscription graph of the app, and they can all affect each other, giving us a useful layer of protection.

cc @J-Son89

Reply:
I added the tests @ilmotta